src/realm/realmReducer.js

```diff
@@ -88,15 +108,28 @@ export default (
         messageContentDeleteLimitSeconds: action.data.realm_message_content_delete_limit_seconds,
         messageContentEditLimitSeconds: action.data.realm_message_content_edit_limit_seconds,
         pushNotificationsEnabled: action.data.realm_push_notifications_enabled,
+        webPublicStreamsEnabled: action.data.server_web_public_streams_enabled ?? false,
+        createWebPublicStreamPolicy: action.data.realm_create_web_public_stream_policy ?? 6,
+        enableSpectatorAccess: action.data.realm_enable_spectator_access ?? false,
 
-        email: action.data.email,
-        user_id: action.data.user_id,
-
-        // TODO(#4933): Use modern `user_settings` object for this
-        twentyFourHourTime: action.data.twenty_four_hour_time,
+        //
+        // InitialDataRealmUser
+        //
 
         canCreateStreams: action.data.can_create_streams,
         isAdmin: action.data.is_admin,
+        isOwner: action.data.is_owner,
+        isModerator: action.data.is_moderator ?? false,
```

Comment:
```suggestion
        isModerator: action.data.is_moderator ?? action.data.is_admin,
```

IIUC `is_moderator` will (on servers with that concept) always be true when `is_admin` is -- it really means "is *at least* a moderator". So this way that invariant continues to hold when we compute this value for older servers.

Reply:
That sounds right; I agree.